projects/components/src/paginator/paginator.component.ts

```diff
@@ -27,12 +28,14 @@ import { PaginationProvider } from './paginator-api';
       </ht-label>
 
       <div class="pagination-buttons">
-        <!-- TODO: Change ToggleButtonComponent to be more generic ButtonGroupComponent to accommodate this use case -->
+        <!-- TODO: Change 
+         to be more generic ButtonGroupComponent to accommodate this use case -->
         <ht-button
```

Comment:
We also have a toggle button component, can we use that here instead?

Reply:
Change ht-button to ht-toggle-button-group